lib/matplotlib/text.py

```diff
@@ -1657,17 +1657,41 @@ def set_figure(self, fig):
 
 
 class OffsetFrom(object):
+    '''Callable helper class for working with `Annotation`
+    '''
     def __init__(self, artist, ref_coord, unit="points"):
+        '''
+        Parameters
+        ----------
+        artist : `Artist`, `BboxBase`, or `Transform`
+            The object to compute the offset from.
+
+        ref_coord : length 2 sequence
+            The location to offset relative to (in fractions of
+            the `artist` bounding box.  If `artist` is a transform,
+            offset relative to the transform applied to this value
```

Comment:
Also, I find this description a bit confusing. After mulling it over, I _think_ what it's saying is that the transform will be calculated using the `ref_coord` _within_ the `artist` as an "origin" (unless `artist` is a Transform; then it's just an extra offset).

I think the problem is that "offset" can be interpreted as either a noun or a verb in both these sentences, so the meaning is unclear depending on how the reader interprets it. Maybe picking a more specific word instead of "offset" will help offset that confusion (okay, sorry for that small pun there.)